dipy/viz/actor.py

```diff
@@ -201,6 +201,142 @@ def copy(self):
     return image_actor
 
 
+def contour_actor(data, affine=None, levels=[50],
+                  colors=[np.array([1.0, 0.0, 0.0])], opacities=[1]):
+    """Take a volume and draw surface contours for any any number of
+    thresholds (levels) where every contour has its own color and opacity
+
+    Parameters
+    ----------
+    data : array, shape (X, Y, Z) or (X, Y, Z, 3)
+        A grayscale or rgb 4D volume as a numpy array.
```

Comment:
Can you really have a contour plot of an RGB volume? The provided `levels` thresholds would correspond to what?